api/config/functions/bootstrap.js

```diff
@@ -1,4 +1,4 @@
-'use strict';
+"use strict";
```

Comment:
Are we using Prettier or any linter? It seems weird to use double quotes instead of single quotes.